### ISSUE TITLE
Error getting a list of isues: 400 Bad Request

### ISSUE THREAD

**Issue by jessesquires** [lowply/auto-closer#3]
Hello! 👋🏼 

When I try to use this action, it fails with the error:

```
Error getting a list of isues: 400 Bad Request
```

This is in a private organization repo -- I'm not sure if that's a problem for the default `GITHUB_TOKEN` ?

My workflow looks like this:

```yml
jobs:
  open:
    name: Open New Meeting Agenda
    runs-on: ubuntu-latest
    steps:
      - name: git checkout
        uses: actions/checkout@v2
      # https://github.com/marketplace/actions/issue-from-template
      - name: issue from template
        uses: lowply/issue-from-template@v0.1.0
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          IFT_TEMPLATE_NAME: meeting_agenda.md
  close:
    name: Close Previous Meetings
    runs-on: ubuntu-latest
    needs: open
    steps:
      # https://github.com/marketplace/actions/auto-closer
      - uses: lowply/auto-closer@v0.0.5
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          AC_LABEL: "meeting agenda"
          AC_KEEP: 1
```

**Comment by lowply** [comment]
Hi @jessesquires thank you so much for catching this, and again I'm sorry for the delay in getting back to you. I've fixed this in https://github.com/lowply/auto-closer/pull/4.

**Comment by jessesquires** [comment]
Thanks @lowply ! 🙌🏼 

Does a new version need to be tagged?

**Comment by lowply** [comment]
@jessesquires oops, you're right, sorry about that! It's now tagged and released as v0.0.6: https://github.com/lowply/auto-closer/releases
